### PR TITLE
feat: updates to mfa styling

### PR DESCRIPTION
**Pull request by YazeedLoonat**
## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/1928#issuecomment-1042525792)

- [x] This change addresses the issue in full

## Description
This should address the styling issues with the mfa pathways 

## How Can This Be Tested/Reviewed?
Go to the partner's site and log in as user with mfa enabled
as you go through the mfa process, verify that the styling matches what is in the mock, and in the comments of the ticket linked above

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by YazeedLoonat** [comment]
This should now also address the changes requested in:
https://github.com/bloom-housing/bloom/issues/1928#issuecomment-1050136761

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
We're still missing the styling fixes for the box (the margin and the border not going all the way across) on the phone number page, on each page we have a gray border-b on the card we should remove. I'm also getting a 500 on phone number entry but we can continue to touch base offline there it might be a personal issue.

**Comment by emilyjablonski** [comment]
Screen Shot 2022-02-24 at 2 55 47 PM
If I enter an incorrect code I get an "email/password" error

File: ui-components/src/global/forms.scss [hunk 1]
```diff
@@ -376,3 +376,7 @@ progress,
   @apply bg-primary;
   transition: width 0.25s;
 }
+
+.field-helper-element {
```
Comment (emilyjablonski) [inline ui-components/src/global/forms.scss:380]: This doesn't seem to be used anywhere

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Some changes to make:
* We still need to remove the bottom border on all the cards - I hover over it at the end of the attached video if that helps!
* We still need to update the error message for an incorrect code to not show invalid email/password
* The edit phone number link is now cursor-pointer and focusable, but it is not actionable on enter and it's not in the right tab order - it is the first focusable element on the page
* If I enter the wrong code and try to enter a new one, the error message pops up again after the first valid keypress


https://user-images.githubusercontent.com/65367387/155635731-8be84a98-0ea4-45f8-8684-6dae245c617a.mov

Let me know if you wanna pair tomorrow!

**Comment by YazeedLoonat** [comment]
Hey @emilyjablonski I think I got all your points now, thank you for the video it really helped me understand what you were pointing out

for `We still need to update the error message for an incorrect code to not show invalid email/password` I left the verbiage in for email and password, but added mfacode verbiage too when its appropriate. I remember from a previous conversation we were unsure if we wanted to obfuscate what was wrong with the login in request or not. I can change that if you think its better to specify that at that point its the code thats wrong

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
We touched base offline, but:

- The borders and error message pop issues resolved and look great
- We're touching base with Jesse offline to confirm the error string
- After the latest set of changes I'm no longer able to change my phone number (below)
- Locking out after 5 failed codes is still letting me through once I enter a correct code (again sorry for not testing this flow earlier :( I didn't think it was meant to lock)

Screen Shot 2022-02-25 at 11 33 43 AM

**Review by emilyjablonski — APPROVED** [review]
Pulling lockout issues into a separate ticket - once Jesse gets back about the error message GTM!

File: ui-components/src/locales/general.json [hunk 2]
```diff
@@ -967,7 +967,7 @@
       "accountHasBeenLocked": "For security reasons, this account has been locked.",
       "youHaveToWait": "You’ll have to wait 30 minutes since the last failed attempt before trying again.",
       "enterValidEmailAndPassword": "Please enter a valid email and password.",
-      "enterValidEmailAndPasswordAndMFA": "Please enter a valid email, password, and code.",
+      "enterValidEmailAndPasswordAndMFA": "Please enter a valid code",
```
Comment (emilyjablonski) [inline ui-components/src/locales/general.json:970]: Sorry to ask for such a small change but all of the other similar error strings end in a period - after that good to merge 🥳

**Comment by YazeedLoonat** [comment]
Hey @seanmalbert could you peep this one, one last time please

**Review by seanmalbert — APPROVED** [review]
This looks good to me, I created [this issue](https://github.com/bloom-housing/bloom/issues/2547), to address partner users not being able to login to the public site.

**Pull request merged**